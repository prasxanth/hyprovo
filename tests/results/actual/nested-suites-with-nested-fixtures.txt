2021-11-16 09:27:35.643 - INFO - Pass ✓... (suite-with-fixtures / test-mul-variable): mul pass
2021-11-16 09:27:35.646 - ERROR - Fail ✗... (suite-with-fixtures / test-mul-variable): mul fail 
                                    Expected: 4
                                    Got: 1
2021-11-16 09:27:35.650 - INFO - Pass ✓... (suite-with-fixtures / test-div-variable): div pass
2021-11-16 09:27:35.652 - ERROR - Fail ✗... (suite-with-fixtures / test-div-variable): div fail 
                                    Expected: 4
                                    Got: 1.0
2021-11-16 09:27:35.657 - ERROR - Fail ✗... (suite-with-fixtures / test-mul-variable): mul pass 
                                    Expected: 1
                                    Got: 6
2021-11-16 09:27:35.660 - ERROR - Fail ✗... (suite-with-fixtures / test-mul-variable): mul fail 
                                    Expected: 4
                                    Got: 6
2021-11-16 09:27:35.665 - ERROR - Fail ✗... (suite-with-fixtures / test-div-variable): div pass 
                                    Expected: 1
                                    Got: 1.5
2021-11-16 09:27:35.667 - ERROR - Fail ✗... (suite-with-fixtures / test-div-variable): div fail 
                                    Expected: 4
                                    Got: 0.6666666666666666
2021-11-16 09:27:35.673 - INFO - Pass ✓... (suite-with-nested-fixture / test-add-variable): add pass
2021-11-16 09:27:35.676 - ERROR - Fail ✗... (suite-with-nested-fixture / test-add-variable): add fail 
                                    Expected: 3
                                    Got: 2
2021-11-16 09:27:35.680 - INFO - Pass ✓... (suite-with-nested-fixture / test-mul-variable): mul pass
2021-11-16 09:27:35.682 - ERROR - Fail ✗... (suite-with-nested-fixture / test-mul-variable): mul fail 
                                    Expected: 4
                                    Got: 1
2021-11-16 09:27:35.682 - DEBUG - (nested-suites-with-nested-fixtures): Test Summary → Total: 12    Pass: 4    Fail: 8
