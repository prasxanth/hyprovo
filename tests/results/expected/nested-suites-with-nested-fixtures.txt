2021-11-14 11:26:50.351 - INFO - Pass ✓... (suite-with-fixtures / test-mul-variable): mul pass
2021-11-14 11:26:50.355 - ERROR - Fail ✗... (suite-with-fixtures / test-mul-variable): mul fail 
                                    Expected: 4
                                    Got: 1
2021-11-14 11:26:50.363 - INFO - Pass ✓... (suite-with-fixtures / test-div-variable): div pass
2021-11-14 11:26:50.367 - ERROR - Fail ✗... (suite-with-fixtures / test-div-variable): div fail 
                                    Expected: 4
                                    Got: 1.0
2021-11-14 11:26:50.375 - ERROR - Fail ✗... (suite-with-fixtures / test-mul-variable): mul pass 
                                    Expected: 1
                                    Got: 6
2021-11-14 11:26:50.379 - ERROR - Fail ✗... (suite-with-fixtures / test-mul-variable): mul fail 
                                    Expected: 4
                                    Got: 6
2021-11-14 11:26:50.388 - ERROR - Fail ✗... (suite-with-fixtures / test-div-variable): div pass 
                                    Expected: 1
                                    Got: 1.5
2021-11-14 11:26:50.393 - ERROR - Fail ✗... (suite-with-fixtures / test-div-variable): div fail 
                                    Expected: 4
                                    Got: 0.6666666666666666
2021-11-14 11:26:50.404 - INFO - Pass ✓... (suite-with-nested-fixture / test-add-variable): add pass
2021-11-14 11:26:50.408 - ERROR - Fail ✗... (suite-with-nested-fixture / test-add-variable): add fail 
                                    Expected: 3
                                    Got: 2
2021-11-14 11:26:50.418 - INFO - Pass ✓... (suite-with-nested-fixture / test-mul-variable): mul pass
2021-11-14 11:26:50.422 - ERROR - Fail ✗... (suite-with-nested-fixture / test-mul-variable): mul fail 
                                    Expected: 4
                                    Got: 1
2021-11-14 11:26:50.422 - DEBUG - (nested-suites-with-nested-fixtures): Test Summary → Total: 12    Pass: 4    Fail: 8
